src/vsphere_cpi/lib/cloud/vsphere/drs_rules/drs_rule.rb

```diff
@@ -32,6 +32,22 @@ def add_vm(vm)
       end
     end
 
+    # Adds VM to given VM Group & creates VM/HOST Affinity rule
+    # @param [Vim::VirtualMachine] vm
+    # @param [String] vm_group_name
+    # @param [String] host_group_name
+    def add_vm_host_affinity_rule(vm, vm_group_name, host_group_name)
```

Comment:
vm is not needed with current implementation. I think we can refactor later to make addition of vm to vm_group as part of this code rather then doing it separately

Reply:
`vm` is removed. I am not sure about merging addition of vm group to this function or I do not understand your intent. What  value would it bring rather than doing this way?

Reply:
To be discussed later for future releases